static/js/redux/ui/modals/user_settings_modal.jsx

```diff
@@ -109,6 +109,7 @@ class UserSettingsModal extends React.Component {
   }
 
   render() {
+    let favorite_num = 10; // bad choice, but temporary anyway
```

Comment:
Task 2 specifies that we want the student's favorite number saved in the database, along with the rest of the student's data. The point of Task 2 is to practice editing Django models, creating Django migrations, and then bringing that value forward to the frontend from the database.

Reply:
I'll figure out how to do this. Thanks for letting me know.